src/components/Newsletter/Newsletter.js

```diff
@@ -46,6 +52,25 @@ class Newsletter extends React.Component {
     }
   }
 
+  async subscribeToAlerts() {
+    const email = this.emailInput.value;
+    const locations = this.alertsSelectionArray.map(
+      item => item.full_fips_code,
+    );
+    const db = getFirebase().firestore();
+    // Merge the locations with any existing ones since that's _probably_ what the user wants.
+    await db
+      .collection('alerts-subscriptions')
+      .doc(email)
+      .set(
+        {
+          locations: firebase.firestore.FieldValue.arrayUnion(...locations),
+          subscribedAt: firebase.firestore.FieldValue.serverTimestamp(),
+        },
+        { merge: true },
```

Comment:
@chasulin FYI- This is the only actual bit of real Firebase / Firestore code.

It just creates a document for the user, containing the subscribed locations and a timestamp.

We'll need to create some sort of "unsubscribe" page as well that deletes users from Firestore.  If you _want_ to work on that, you could.  See my overall PR comment.